corehq/apps/export/utils.py

```diff
@@ -165,6 +165,10 @@ def convert_saved_export_to_export_instance(domain, saved_export, dryrun=False):
                     ))
 
                 new_column = _get_normal_column(new_table, column_path, transform)
+
+                # If we can't find the column in the current table
+                # look in every other table to see if the column is a repeat
+                # that did not receive more than one entry
```

Comment:
yeah, but if a normal repeat doesn't register as a repeat then you only need to check the main table, no? i figured checking all tables must be because of nested repeats